strings/en/levels.json

```diff
@@ -3,6 +3,5 @@
 	"pageTitle": "Here is the XP leaderboard!",
 	"level": "#%%rank%% | Level %%level%% | XP: %%xp%%",
 	"youIndicator": "YOU",
-	"thousand": "k",
```

Comment:
Please add this one (and only this one) back in order to keep the translations on crowdin

Reply:
Resolved in latest commit